docs/source/tensors.rst

```diff
@@ -42,7 +42,12 @@ A tensor can be constructed from a Python :class:`list` or sequence using the
      1 -1
     [torch.FloatTensor of size (2,2)]
 
-An tensor of specific data type can be constructed by passing a
+.. warning::
+
+    If :attr:`data` is a :class:`torch.Tensor`, the output tensor
```

Comment:
The output is always a copy of the data independent of the type of the input; I agree it might be most surprising for Tensors but this difference exists for e.g. numpy arrays too.

Reply:
yes, but in numpy we have a `copy` kwarg that allows us to avoid the copy if possible

Reply:
Okay, will include other sequence types in the warning message